ExposureNotification.App/ExposureNotification.App/ExposureNotification.App.csproj

```
@@ -15,6 +15,10 @@
 		<EmbeddedResource Include="Resources\Fonts\fa-brands-400.ttf" />
 		<EmbeddedResource Include="Resources\Fonts\fa-regular-400.ttf" />
 		<EmbeddedResource Include="Resources\Fonts\fa-solid-900.ttf" />
+		<EmbeddedResource Include="settings.json" Condition="'$(Configuration)' == 'Debug'" />
```

```suggestion
		<EmbeddedResource Include="settings.json" Condition="'$(Configuration)' == 'Debug' or !Exists('local.settings.json')" />
```

Include the committed file if there is no local copy - as would be on a clean checkout.